tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -170,6 +170,27 @@ def absolute_call_i(x):
     assert(isclose(f1(-x), absolute_call_i(-x), rtol=RTOL, atol=ATOL))
     assert matching_types(f1(x), absolute_call_i(x))
 
+def test_absolute_call_c(language):
+    @template(name='T', types=['complex','complex64','complex128'])
+    @types('T')
+    def absolute_call_c(x):
+        from numpy import absolute
+        return absolute(x)
+
+    f1 = epyccel(absolute_call_c, language = language)
+    x = uniform(high=1e6)+1j*uniform(high=1e6)
+    assert(isclose(f1(x), absolute_call_c(x), rtol=RTOL, atol=ATOL))
+    assert(isclose(f1(-x), absolute_call_c(-x), rtol=RTOL, atol=ATOL))
+    assert matching_types(f1(x), absolute_call_c(x))
+
+    x = np.complex64(uniform(high=1e6)-1j*uniform(high=1e6))
+    assert(isclose(f1(x), absolute_call_c(x), rtol=RTOL, atol=ATOL))
+    assert matching_types(f1(x), absolute_call_c(x))
+
+    x = np.complex128(uniform(high=1e6)-1j*uniform(high=1e6))
+    assert(isclose(f1(x), absolute_call_c(x), rtol=RTOL, atol=ATOL))
+    assert matching_types(f1(x), absolute_call_c(x))
+
```

Comment:
Why do we need to use `isclose` to verify that the absolute value is correct? Assuming that the IEEE 754 standard is respected by all implementations, I would expect all digits to be correct after calling `abs`.

Reply:
Sorry. I have just realized that these tests are for complex numbers only. It makes sense to use a tolerance for comparison, then!

Reply:
The question remains for testing the functions `absolute_call_r` and `absolute_call_i`. But this not strictly related to this PR.